leftwm/src/config/keybind.rs

```diff
@@ -36,6 +36,8 @@ impl Keybind {
             }
             BaseCommand::FocusWindowTop if value_is_some => {
                 bool::from_str(&self.value).context("invalid boolean value for FocusWindowTop")?;
+            BaseCommand::FocusNextTag => leftwm_core::Command::FocusNextTag,
+            BaseCommand::FocusPreviousTag => leftwm_core::Command::FocusPreviousTag,
```

Comment:
@AethanFoot I dropped these two lines with `4e9cfd3` as I had no idea, where to move them to.
So far everything is working fine here on my machine.

Reply:
Ye that is right as the match statement is just used to check whether a command which needs a value has a valid one.